views/lib/extended_footer.hbs

```diff
@@ -45,10 +44,9 @@
 					</div>
 					-->
 				</div>
-				<h2 class="h4 card-title">Stellenausschreibungen</h2>
+				<h2 class="h4 card-title">{{$t "lib.extended_footer.heading.vacancies" }}</h2>
 				<div class="form-group">
-					Du möchtest gerne Mitglied in unserem <span style="white-space: nowrap;">HPI Schul-Cloud</span> Team werden?
-					Wir suchen:
+					{{$t "lib.extended_footer.text.youWouldLikeToBeAMember" }} <span style="white-space: nowrap;">HPI Schul-Cloud</span> {{$t "lib.extended_footer.text.becomeATeam" }}
```

Comment:
merge this into one {{{$t "lib.extended_footer.text.weAreHiring(orsomething)" }}}
.json:
"weAreHiring" :  "Du möchtest gerne Mitglied in unserem <span style=\"white-space: nowrap;\">HPI Schul-Cloud</span> Team werden? Wir suchen:"